### PR TITLE
Stampy redesign 591

### PR DESCRIPTION
**Pull request by buddy-web3**
#591 #590

### CODE REVIEW

File: app/components/Chatbot/index.tsx [hunk 1]
```diff
@@ -248,7 +249,7 @@ export const Chatbot = ({question, questions}: {question?: string; questions?: s
   })
 
   return (
-    <div className="centered col-9 padding-bottom-128">
+    <div className="centered col-9 height-70">
```
Comment (melissasamworth) [inline app/components/Chatbot/index.tsx:252]: I believe this should take up 10 columns
image

Comment (melissasamworth) [reply app/components/Chatbot/index.tsx:252]: If you want to leave this to me when I do css that's totally fine

File: app/components/Chatbot/widgit.css [hunk 2]
```diff
@@ -20,3 +20,29 @@
     width: 100%;
   }
 }
+.height-70 {
```
Comment (melissasamworth) [inline app/components/Chatbot/widgit.css:23]: oh interesting. For now on we could do .height-70vh so we know the units!

File: app/components/Chatbot/widgit.css [hunk 3]
```diff
@@ -20,3 +20,29 @@
...
+}
+.warning-floating {
+  position: fixed;
+  right: 7vw;
+  z-index: 100;
+  bottom: 5vw;
+  width: 11vw;
+}
+.red {
+  color: #d40000;
```
Comment (melissasamworth) [inline app/components/Chatbot/widgit.css:47]: We'll want to define this and reference it as a variable, plus put this class in the same place as the other color classes in root.css

File: app/components/Nav/index.tsx [hunk 4]
```diff
@@ -28,6 +29,12 @@ export const Nav = ({toc, categories}: NavProps) => {
             id="showArticles"
           />
           <ArticlesDropdown toc={toc} categories={categories || []} />
+          <MenuItem
+            primary={true}
+            link="https://chat.aisafety.info"
+            icon={<BotIcon />}
+            text="Stampy chatbot"
```
Comment (melissasamworth) [inline app/components/Nav/index.tsx:36]: This is the global nav right? Could we make it say "AI Safety chatbot" instead? :)

**Review by melissasamworth — CHANGES_REQUESTED** [review]

**Comment by buddy-web3** [comment]
@melissasamworth "If you want to leave this to me when I do css that's totally fine" - yes please, take over the design parts. Please note that all of those are unrelated to the tickets.. they are some design changes that I had stashed and now somehow got pushed.

**Review by melissasamworth — APPROVED** [review]

**Pull request merged**